ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -4,7 +4,7 @@ This folder contains the files to configure IPsec encryption for external traffi
 
 * Enable IPsec encryption yaml
 * IPsec endpoint configuration with NMState yml
-* A MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
+* build.sh script to create a MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
 
 Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
 
```

Comment:
Lets add the following text here to provide some more context about the different installation options:
It is possible to enable IPSec and install the configuration and certificate with ZTP (Day 1, before operator installations) or manually, once the SNO cluster has already been deployed. Instructions will be provided below for both options.